sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.cs

```diff
@@ -27,6 +28,56 @@ internal static TextAnalyticsError ConvertToError(TextAnalyticsErrorInternal err
             return new TextAnalyticsError(errorCode, message, target);
         }
 
+        internal static List<TextAnalyticsError> ConvertToErrors(IReadOnlyList<TextAnalyticsErrorInternal> internalErrors)
+        {
+            var errors = new List<TextAnalyticsError>();
+
+            if (internalErrors == null)
+            {
+                return errors;
+            }
+
+            foreach (TextAnalyticsErrorInternal error in internalErrors)
+            {
+                errors.Add(ConvertToError(error));
+            }
+            return errors;
+
+        }
+
+        internal static IReadOnlyList<KeyPhraseExtractionTasksItem> ConvertToKeyPhraseExtractionTasks(IReadOnlyList<KeyPhraseExtractionTasksItem> keyPhraseExtractionTasks, IDictionary<string, int> idToIndexMap)
+        {
+            var collection = new List<KeyPhraseExtractionTasksItem>();
+            foreach (KeyPhraseExtractionTasksItem task in keyPhraseExtractionTasks)
+            {
+                collection.Add(new KeyPhraseExtractionTasksItem(task, idToIndexMap));
+            }
+
+            return collection;
+        }
+
+        internal static IReadOnlyList<EntityRecognitionPiiTasksItem> ConvertToEntityRecognitionPiiTasks(IReadOnlyList<EntityRecognitionPiiTasksItem> entityRecognitionPiiTasks, IDictionary<string, int> idToIndexMap)
+        {
+            var collection = new List<EntityRecognitionPiiTasksItem>();
+            foreach (EntityRecognitionPiiTasksItem task in entityRecognitionPiiTasks)
+            {
+                collection.Add(new EntityRecognitionPiiTasksItem(task, idToIndexMap));
+            }
+
+            return collection;
+        }
+
+        internal static IReadOnlyList<EntityRecognitionTasksItem> ConvertToEntityRecognitionTasks(IReadOnlyList<EntityRecognitionTasksItem> entityRecognitionTasks, IDictionary<string, int> idToIndexMap)
```

Comment:
could you move this out of the common region? maybe create a `# region analyze`?